supernode/config/config.go

```diff
@@ -22,6 +22,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/dragonflyoss/Dragonfly/pkg/model"
```

Comment:
Could you reorg the import sequence here? Thanks.

Reply:
Thanks for pointing out this. I think I have addressed all these similar nits now.